Doc/library/time.rst

```diff
@@ -772,7 +772,7 @@ These constants are used as parameters for :func:`clock_getres` and
    Similar to :data:`CLOCK_MONOTONIC`, but provides access to a raw
    hardware-based time that is not subject to NTP adjustments.
 
-   Availability: Linux 2.6.28 or later.
+   Availability: Availability: Linux 2.6.28 or later, macOS 10.12 and newer.
```

Comment:
"Availability: Availability:" hum, it's written twice :-) While we are here, a new ".. availability::" markup has been added recently.

Can you please replace "Availability:" with ".. availability::"?

I'm not a native english speaker. What's the best: "<version> or later" or "<version> and newer"? os.rst uses "or newer" 3 times, but "and later" is not used. Maybe replace also "Linux 2.6.28 or later" with "Linux 2.6.28 and newer". Ah, os.rst uses "or newer" and "and newer" :-)

-- Victor the nitpicker

Reply:
> Can you please replace "Availability:" with ".. availability::"?

All "Availability:" in this file, not only this one, please ;-)

Reply:
@vstinner I have seen the use of both "or newer" and "and newer" in the docs. Is this based on semantics or it is just based on preference?